buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -326,7 +345,7 @@ class VagrantTestPlugin implements Plugin<Project> {
                 args '--provision', '--provider', 'virtualbox'
                 /* It'd be possible to check if the box is already up here and output
                   SKIPPED but that would require running vagrant status which is slow! */
-                dependsOn update
+                dependsOn destroy, update
```

Comment:
Update needs to depend on destroy or it could run before destroy.

Reply:
The ordering between `destroy` and `update` does not matter, but I can see value in there being a definite well-defined ordering.

Reply:
I pushed c9aeb4857c8b6bab6c75a328c5b60f23827365fe.